frontend/src/lib/components/DefinitionPopup/definitionPopupLogic.ts

```diff
@@ -24,8 +43,173 @@ export const definitionPopupLogic = kea<definitionPopupLogicType<DefinitionPopup
                 setPopupState: (_, { state }) => state,
             },
         ],
+        localDefinition: [
+            {} as Partial<TaxonomicDefinitionTypes>,
+            {
+                setDefinition: (_, { item }) => item,
+                setLocalDefinition: (state, { item }) => ({ ...state, ...item } as Partial<TaxonomicDefinitionTypes>),
+            },
+        ],
     },
+    loaders: ({ values, props }) => ({
+        definition: [
+            {} as Partial<TaxonomicDefinitionTypes>,
+            {
+                setDefinition: ({ item }) => item as TaxonomicDefinitionTypes,
+                handleSave: async (_, breakpoint) => {
+                    await breakpoint(100)
```

Comment:
Why wait?

Reply:
true true, the button will be disabled anyways when this is triggered